Pizza Divider.js

```diff
@@ -0,0 +1,14 @@
+const submitButton = document.getElementById('submit');
+submitButton.addEventListener('click', function(event) {
+let peopleNumber = document.getElementById('people').value;
+let pizzaNumber = document.getElementById('pizzas').value;
+let sliceNumber = document.getElementById('slices').value;
+let pizzaSlices = sliceNumber * pizzaNumber;
+let fullTotal = pizzaSlices / peopleNumber;
+let sharedSlices = Math.floor(fullTotal);
+let reminderSlices = fullTotal - sharedSlices;
+Math.round(reminderSlices *= peopleNumber);
+let result = `You have ${sharedSlices} each and ${reminderSlices} left to fight over!`;
+const target = document.getElementById('answer');
+target.innerHTML = result;
+})
```

Comment:
What do you think about leaving some empty lines in reasonable places so that the code is more readable?

For example, separate the button targeting from the event listener block with an empty line:

```js
const submitButton = document.getElementById('submit');

submitButton.addEventListener('click', function(event) {
// ... etc
```

Separate input retrieval from the actual calculations with an empty line:

```js
// ...
let sliceNumber = document.getElementById('slices').value;

let pizzaSlices = sliceNumber * pizzaNumber;
// ...etc
```

Separate calculations from preparation of the output with an empty line:

```js
// ...
Math.round(reminderSlices *= peopleNumber);

let result = `You have ${sharedSlices} each and ${reminderSlices} left to fight over!`;
// ...etc
```

That way you separate the three concerns of your code:

1. retrieve inputs
1. calculations
1. prepare output

Reply:
That makes good sense and looks way easier to read. Thanks!